05/DB.test.js

```diff
@@ -1 +1,128 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('.insert(data)', () => {
+  it('return 1 when one item inserted', async () => {
+    const db = new DB();
+    await db.insert({ id: 1, name: 'element 01' });
+    const rows = await db.getRows();
+
+    expect(rows.length).toBe(1);
+  });
+
+  it('throw error when id is duplicated', async () => {
+    expect.assertions(1);
+
+    const db = new DB();
+    try {
+      await db.insert({ id: 2, name: 'element 02' });
+      await db.insert({ id: 2, name: 'element 03' });
+    } catch (err) {
+      expect(err).toBe("ID can't be duplicated!");
+    }
+  });
+
+  it('throw error when id is not a number', async () => {
+    expect.assertions(1);
+
+    const db = new DB();
+    try {
+      await db.insert({ id: '4', name: 'element 04' });
+    } catch (err) {
+      expect(err).toBe('ID can be only number!');
+    }
+  });
+
+  it('iserted data is correct', async () => {
+    const data = { id: 5, name: 'element 05' };
+    const db = new DB();
+    const insertedData = await db.insert(data);
+    expect(insertedData).toEqual(data);
+  });
+});
+
+describe('.select(id)', () => {
+  it('return ID of selected row', async () => {
+    const data = { id: 6, name: 'element 06' };
+    const db = new DB();
+    await db.insert(data);
+
+    expect(data.id).toBe(6);
```

Comment:
Tutaj nie sprawdzasz czy dane są w bazie tylko czy obiekt, który napisałaś ma to id co trzeba.
Tutaj raczej należało wyszukać ten obiekt w tablicy - może też sprawdzić ile tych elementów jest w bazie.

PS. Jeśli chcesz sprawdzić co jest zwracane to powinnaś napisać tak: `const result = await db.insert(data);` i wtedy zmienną `result` wrzucić do `expect`